packages/adblocker/test/parsing.test.ts

```diff
@@ -2648,3 +2649,11 @@ describe('scriptlets arguments parsing', () => {
     }
   });
 });
+
+describe('Domains', () => {
+  it('rejects to parse invalid pipes', () => {
+    for (const str of ['|foo.com', 'foo.com|', '|foo.com|']) {
```

Comment:
shouldn't we just drop `|` and accept `foo.com`?

Reply:
maybe we should support `$to=example.com|||||ghostery.com` as well

Reply:
@chrmod We have tests that defines those cases as an invalid. Please, choose the behavior on this. Personally, I don't feel it's good to accept those typos.

```
  1) #parseFilters
       with not supported filters
         returns invalid network filters:

      AssertionError: expected [] to deeply equal [ { filter: '$domain=|a', …(2) } ]
      + expected - actual

      -[]
      +[
      +  {
      +    "filter": "$domain=|a"
      +    "filterType": 1
      +    "lineNumber": 0
      +  }
      +]
      
      at Context.<anonymous> (test/lists.test.ts:210:24)
      at process.processImmediate (node:internal/timers:491:21)
```